content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -525,10 +525,26 @@ volumeBindingMode: WaitForFirstConsumer
 parameters:
   arrayID: "GlobalUniqueID"
   csi.storage.k8s.io/fstype: "xfs"
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "powerstore-nfs"
+provisioner: "csi-powerstore.dellemc.com"
+parameters:
+  arrayID: "Unique"
+  csi.storage.k8s.io/fstype: "nfs"
+  nasName: "nas-server1, nas-server2, nas-server3"
+  allowRoot: "false"
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
 ```
 
 Here we specify two storage classes: one of them uses the first array and `ext4` filesystem, and the other uses the second array and `xfs` filesystem.
 
+> ℹ️ **NOTE:** : Storage Class for NFS Platform ,users can specify one or multiple NAS servers, separated by commas.
```

Comment:
NFS platform or NFS SC?
`NFS Platform , -> NFS SC, ` i.e. space after , not before?